### PR TITLE
 Fixes #4984 - Remove maximize from Meter (Metro) menu 

### LINKED ISSUE #4984: The small "Metro" window is maximized and can be resized.
This I mentioned in my other thread #4978. and is that I do not see the reason to add a maximize button and the possibility of resizing a window with so few options. while other windows like the mixer if you need them.

ezgif com-video-to-gif (3)

### PR DESCRIPTION
**Pull request by malcops**
Tested locally that the Meter menu does not have a maximize button, and cannot be resized.

First time contributor, and new to Qt.. any feedback is welcome of course.

### CODE REVIEW

**Comment by amazing-username** [comment]
Hey @malcops, funny enough I was working on the same issue as a first-time contributor. I fixed the sizing issue and was going to work on preventing the Meter window from being maximized but forgot the issue # then checked and to see that you made the commit.

**Comment by mattdeckard** [comment]
This fixes #4984 , no?

**Comment by malcops** [comment]
@mattdeckard yes it fixes #4984

File: src/gui/widgets/TempoSyncKnob.cpp [hunk 1]
```diff
@@ -291,7 +291,11 @@ void TempoSyncKnob::showCustom()
 	if( m_custom == NULL )
 	{
 		m_custom = new MeterDialog( gui->mainWindow()->workspace() );
-		gui->mainWindow()->addWindowedWidget( m_custom );
+		m_subWindow = gui->mainWindow()->addWindowedWidget( m_custom );
+		Qt::WindowFlags flags = m_subWindow->windowFlags();
+		flags &= ~Qt::WindowMaximizeButtonHint;
+		m_subWindow->setWindowFlags( flags );
+        m_subWindow->setFixedSize( m_subWindow->size() );
```
Comment (PhysSong) [inline src/gui/widgets/TempoSyncKnob.cpp]: According to our coding conventions, 2 tabs should be used instead of 8 spaces.

Comment (malcops) [reply src/gui/widgets/TempoSyncKnob.cpp]: Fixup commit changes these spaces to tabs. Thanks!

**Comment by DomClark** [comment]
I don't think there's a need to keep a pointer to the subwindow in an `m_subWindow` member. It can stay local to `showCustom`.

File: src/gui/widgets/TempoSyncKnob.cpp [hunk 2]
```diff
@@ -291,7 +291,12 @@ void TempoSyncKnob::showCustom()
 	if( m_custom == NULL )
 	{
 		m_custom = new MeterDialog( gui->mainWindow()->workspace() );
-		gui->mainWindow()->addWindowedWidget( m_custom );
+		QMdiSubWindow * m_subWindow;
```
Comment (BaraMGB) [inline src/gui/widgets/TempoSyncKnob.cpp]: You should remove the "m_" prefix since this is not a member variable.

Comment (PhysSong) [reply src/gui/widgets/TempoSyncKnob.cpp]: Also, you can consolidate the declaration and the assignment as well.

**Comment by DomClark** [comment]
Can this be merged now, or does it still need testing? Also, the new code doesn't quite follow the coding conventions on spaces within parentheses, but does match the surrounding style.

@malcops If you edit the initial comment here to include the words "fixes #4984", GitHub will automatically close that issue when this pull request is merged.

**Pull request merged**